### PR TITLE
[Metrics] Support tablet level metrics

### PR DESCRIPTION
**Pull request by acelyc111**
## Proposed changes

Sometimes we want to detect the hotspot of a cluster, for example, hot scanned tablet, hot wrote tablet, but we have no insight about tablets in the cluster.
This patch introduce tablet level metrics to help to achieve this object, now support 4 metrics on tablets: `query_scan_bytes `, `query_scan_rows `, `flush_bytes `, `flush_count `. However, one BE may holds hundreds of thousands of tablets, so I add a parameter for the metrics HTTP request, and not return tablet level metrics by default.

ref https://github.com/apache/incubator-doris/issues/3517 https://github.com/apache/incubator-doris/issues/4027
## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

-

### CODE REVIEW

File: be/src/http/action/metrics_action.cpp [hunk 1]
```diff
@@ -34,13 +34,14 @@ namespace doris {
 
 void MetricsAction::handle(HttpRequest* req) {
     const std::string& type = req->param("type");
+    const std::string& with_tablet = req->param("with_tablet");
```
Comment (morningman) [inline be/src/http/action/metrics_action.cpp:37]: Need to update the doc `administrator-guide/operation/monitor-metrics/be-metrics.md`

**Comment by morningman** [comment]
UT compile failed

**Comment by acelyc111** [comment]
> UT compile failed

Fixed, but I found some UT which are disabled ever, now will execute and fail.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**